<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDC Partial Directed Coherence estimation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="filesBox">
            <property name="title">
             <string>Открыть файлы:</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QGridLayout" name="filesGridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="file1NameLabel">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="openFile1Button">
                 <property name="text">
                  <string>Открыть...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="estimateBox">
            <property name="title">
             <string>Выполнить:</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QPushButton" name="estLSButton">
                 <property name="text">
                  <string>Рассчитать к-ты АР-модели</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="estPDCButton">
                 <property name="text">
                  <string>Рассчитать PDC</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QCheckBox" name="estChBox">
                 <property name="text">
                  <string>Считать PDC авт.</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="paramsBox">
          <property name="title">
           <string>Задать параметры:</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0" colspan="2">
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="dimensionFromLabel">
                 <property name="text">
                  <string>Размерность от</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="dimensionFromEdit">
                 <property name="text">
                  <string>4</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="dimensionToLabel">
                 <property name="text">
                  <string>до</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QLineEdit" name="dimensionToEdit">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QLabel" name="dimensionIncLabel">
                 <property name="text">
                  <string>ч/з</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <widget class="QLineEdit" name="dimensionIncEdit">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="shiftFromLabel">
                 <property name="text">
                  <string>Задержка от</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="shiftFromEdit">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="shiftToLabel">
                 <property name="text">
                  <string>до</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QLineEdit" name="shiftToEdit">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>ч/з</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="5">
                <widget class="QLineEdit" name="shiftIncEdit">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="freqLabel">
                 <property name="text">
                  <string>Частоты от</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="freqFromEdit">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="freqToLabel">
                 <property name="text">
                  <string>Гц, до</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QLineEdit" name="freqToEdit">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="4">
                <widget class="QLabel" name="vibLabel">
                 <property name="text">
                  <string>Гц, оцифровка</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="5">
                <widget class="QLineEdit" name="vibEdit">
                 <property name="text">
                  <string>1000</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="arCoeffsLabel">
               <property name="text">
                <string>Коэффициенты АР модели:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QTabWidget" name="arCoeffsTabs">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="tab">
                <attribute name="title">
                 <string>Tab 1</string>
                </attribute>
               </widget>
               <widget class="QWidget" name="tab_2">
                <attribute name="title">
                 <string>Tab 2</string>
                </attribute>
               </widget>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="lagsLabel">
               <property name="text">
                <string>Лаги:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="shiftsLabel">
               <property name="text">
                <string>Сдвиги:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QTableWidget" name="lagsTableWidget">
               <property name="cornerButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QTableWidget" name="shiftsTableWidget"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="statusLabel1">
          <property name="text">
           <string>Статус:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statusLabel2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
